metadata/src/test/java/org/apache/kafka/controller/PartitionChangeBuilderTest.java

```diff
@@ -389,7 +389,7 @@ public void testNoLeaderEpochBumpOnIsrExpansionDuringMigration(String metadataVe
      * always results in a leader epoch increase.
      */
     @ParameterizedTest
-    @ValueSource(strings = {"3.4-IV0", "3.5-IV2", "3.6-IV0", "3.7-IV2", "3.9-IV1"})
+    @ValueSource(strings = {"3.4-IV0", "3.5-IV2", "3.6-IV0", "3.7-IV2", "4.0-IV0"})
```

Comment:
I had a followup comment in https://github.com/apache/kafka/pull/16347#discussion_r1661558720. Do we need to add 4.0-IV0 for `testNoLeaderEpochBumpOnEmptyTargetIsr`?

Reply:
It would not make sense to add 4.0-IV0 to `testNoLeaderEpochBumpOnEmptyTargetIsr`, since in 4.0 an empty target ISR actually WILL trigger a leader epoch bump.

The reason is because once KIP-966 is implemented, empty ISRs will actually be possible, whereas they are not prior to KIP-966